### ISSUE TITLE
broken `brave://downloads` & `brave://extensions` pages

### ISSUE THREAD

**Issue by petercao** [brave/brave-browser#37859]
brave nightly lack upgrade button in brave://extensions/ page

brave release version is ok about above issue;
only happend to nightly version, 
wish you can fix this asap!

**Comment by rebron** [comment]
@petercao Are you talking about the Update button when selecting Developer mode or something else?
<img width="1410" alt="Screenshot 2024-04-26 at 3 48 42 PM" src="https://github.com/brave/brave-browser/assets/5951041/aa118854-eac1-459e-91dd-0c233e44e237">

**Comment by LaurenWags** [comment]
1.65.x is fine, 1.67.x is fine, but 1.66.x is not - maybe related to some Nala changes @rebron?

1.65.x | 1.66.x | 1.67.x
--- | --- | ---
<img width="1205" alt="Screenshot 2024-05-08 at 3 12 29 PM" src="https://github.com/brave/brave-browser/assets/28145373/b676dcbd-3672-42e1-a5a8-c5cb922feccb"> | <img width="1193" alt="Screenshot 2024-05-08 at 3 12 24 PM" src="https://github.com/brave/brave-browser/assets/28145373/0ab8af19-50d4-44e5-bcd1-01a4b9d89b7b"> | <img width="1258" alt="Screenshot 2024-05-08 at 3 16 09 PM" src="https://github.com/brave/brave-browser/assets/28145373/96e869a5-0741-4271-ab4e-0427fd2646d9">

The buttons are still there on 1.66.x, but they're so big, you need to scroll just in that very top area. There's a scroll bar behind the "Developer mode" toggle, but it's hard to see:


https://github.com/brave/brave-browser/assets/28145373/c55f5229-f3e1-4f25-8445-7b7d18328f93

**Comment by fallaciousreasoning** [comment]
Oh no... Hmmm, I'll see if I can work out what fixed it.

**Comment by fallaciousreasoning** [comment]
I think this PR fixed it: https://github.com/brave/brave-core/pull/23276. I'm creating an uplift

**Comment by kjozwiak** [comment]
@brave/qa-team re: the STR/Cases, ensure that both `brave://extensions` & `brave://downloads` is working/being displayed as expected. An example of a broken `brave://downloads` can be found via https://github.com/brave/brave-core/pull/23276#issue-2262387665.

**Comment by kjozwiak** [comment]
The above requires `1.66.105` or higher for `1.66.x` verification 👍

**Comment by GeetaSarvadnya** [comment]
Verification **PASSED** on

```
Brave | 1.66.105 Chromium: 125.0.6422.41 (Official Build) (64-bit)
-- | --
Revision | ba88d7bb175a55287d14014823d5da31fe9c8a47
OS | Windows 10 Version 22H2 (Build 19045.4291)
```

Reproduced the issue on `1.66.97 Chromium: 124.0.6367.118`

brave://extensions/ | Download broken
----------------------|-----------------------
![image](https://github.com/brave/brave-browser/assets/38657976/8271f94d-5d71-45d3-a98a-0bdad8e0ec19) | ![image](https://github.com/brave/brave-browser/assets/38657976/c892d7a4-0810-416e-b750-96b01b9f4a4b)


Confirmed that issue is fixed and working fine on `1.66.105`.Using the examples re: broken states of both `brave://extenstions` & `brave://downloads` from https://github.com/brave/brave-core/pull/23276#issue-2262387665 & https://github.com/brave/brave-browser/issues/37859#issuecomment-2101269295, ensured that everything is being displayed correctly as per the following:

**`Example`** | **`Example`**
---------------|------------------
![image](https://github.com/brave/brave-browser/assets/38657976/2cf1968c-bd7b-499a-8fba-c61e4bc99c0d) | ![image](https://github.com/brave/brave-browser/assets/38657976/63f0ce97-4130-4530-928d-4593146c9fa0)
